src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class declaration should not begin with an empty line. Please remove the empty line above the class declaration.